### PR TITLE
EditUser operable, transaction logic operable, choice Wip

### PR DESCRIPTION
**Pull request by MichalMichanski**
Now that we know where's what edit user is all good, i will seek feedback regarding the references , you might notice a bit of a mess under edit user position in main menu, let me know if im wrong please, eventually ill ask the trainers

In progress - choosing which transaction to edit - any ideas how to go about it without unique values ;_; ? 
Logic of the edit itself is finished

### CODE REVIEW

**Comment by Mmicboop** [comment]
I put more than i'd like into menu as i needed to use methods both from viewers and console factory, and didnt want to let everything access everything, not sure if its acceptable practice, feedback super duper welcome, refactor next week the soonest tho cause i havent even touched the hr and homework yet due to being a procrastinating retard.

File: d [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[33mc01b6cb[m[33m ([m[1;36mHEAD -> [m[1;32mDevelop[m[33m, [m[1;31morigin/Develop[m[33m)[m Revert "Merge pull request #4 from infoshareacademy/bt-7"
```
Comment (MaciejCVsGit) [inline d:1]: What is this? and what is it for? comments?

File: TheBTeam.ConsoleApp/ConsoleFactory.cs [hunk 2]
```diff
@@ -337,6 +357,122 @@ public static Transaction ApplyTransaction(Transaction transaction, User user)
...
+                            Console.WriteLine("Select X to abort or S to save please\n");
+                        }
+                    }
+                }
+            }
+            return tempuser;
+        }
+        public static Transaction EditTransaction(Transaction transaction)
+        {
+            var tempTransaction = new Transaction(transaction.User, transaction.Type, transaction.Category, transaction.Currency, transaction.Amount);
```
Comment (MaciejCVsGit) [inline TheBTeam.ConsoleApp/ConsoleFactory.cs:436]: Moim zdaniem efektywniej było by robić zmiany bezpośrednio na transaction, ale jest to ciekawy pomysł z tymczasową zmienną transaction

**Review by MaciejCVsGit — CHANGES_REQUESTED** [review]

File: d (hunk 1 above)
Comment (MichalMichanski) [reply d:1]: lmao i have no clue, is this gitlog? no idea why its here

**Pull request merged**